### PR TITLE
Mongdb 7 ledger node 11

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Review by aljones15 — APPROVED** [review]
I'm assuming this has been tested etc.

File: lib/http.js [hunk 1]
```diff
@@ -115,7 +114,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
             if(!req.user) {
               return callback(null, []);
             }
-            const {actor} = req.user || {};
+            const {actor} = req.user;
```
Comment (gannan08) [inline lib/http.js:117]: Is there code in another part of the stack that guarantees the existence of `req.user`?

Comment (mattcollier) [reply lib/http.js:117]: Yes, 3 lines above this on L114.  :)

**Review by gannan08 — APPROVED** [review]

**Review by dlongley — APPROVED** [review]
Thanks! Rubberstamp applied.

**Pull request merged**